l10n/it.json

```diff
@@ -0,0 +1,83 @@
+	"Home" : "Pagina iniziale",
+	"participated" : "Partecipazioni",
+	"Yourself" : "Te",
+	"Select" : "Seleziona",
+	"Close" : "Chiudi",
+	"Users" : "Utenti",
+	"Groups" : "Gruppi",
+	"Do you really want to delete that poll?" : "Vuoi davvero cancellare questo sondaggio?",
+	"You must enter at least a title for the new poll." : "Devi inserire almeno il titolo per il nuovo sondaggio.",
+	"Copy to clipboard: Ctrl+C, Enter" : "Copia: Ctrl+C, Invio",
+	"Thanks that you've voted. You can close the page now." : "Grazie per aver votato. Ora puoi chiudere la pagina.",
+	"Nothing selected!\nClick on cells to turn them green." : "Nessuna selezione!\nClicca sulle celle per farle diventare verdi.",
+	"You are not registered.\nPlease enter your name to vote\n(at least 3 characters!)" : "Non sei registrato.\nPer favore, inserisci il tuo nome per votare\n(almeno 3 caratteri!)",
+	"You already have an item with the same text" : "Esiste già una voce di sondaggio con lo stesso testo",
+	"Please choose the groups or users you want to add to your poll." : "Per favore, scegli i gruppi o gli utenti a cui vuoi consentire di votare per questo sondaggio.",
+	"Click to get link" : "Clicca per ottenere il collegamento",
+	"Edit access" : "modifica l'accesso",
+	"Type" : "Tipo",
+	"Event schedule" : "Pianificazione evento",
+	"Text based" : "Lista di elementi",
+	"Text item" : "Testo",
+	"Dates" : "Date",
+	"Description (will be shown as tooltip on the summary page)" : "Descrizione (verrà mostrata come tooltip sulla pagina principale)",
+	"Poll expired" : "Sondaggio scaduto",
+	"The poll expired on %s. Voting is disabled, but you can still comment." : "Il sondaggio è scaduto il %s. Il voto è disabilitato, ma è ancora possibile aggiungere commenti.",
+	"You are not allowed to view this poll or the poll does not exist." : "Non hai i permessi per visualizzare questo sondaggio oppure il sondaggio non esiste.",
+	"Error" : "Errore",
+	"Hello %s,<br/><br/><strong>%s</strong> shared the poll '%s' with you. To go directly to the poll, you can use this link: <a href=\"%s\">%s</a>" : "Ciao %s,<br/><br/><strong>%s</strong> ha condiviso il sondaggio '%s' con te. Vai direttamente al sondaggio con il seguente collegamento: <a href=\"%s\">%s</a>",
+	"Hello %s,<br/><br/><strong>%s</strong> commented on the poll '%s'.<br/><br/><i>%s</i><br/><br/>To go directly to the poll, you can use this link: <a href=\"%s\">%s</a>" : "Ciao %s,<br/><br/><strong>%s</strong> ha commentato il sondaggio '%s'.<br/><br/><i>%s</i><br/><br/> Vai direttamente al sondaggio con il seguente collegamento: <a href=\"%s\">%s</a>",
+	"Hello %s,<br/><br/><strong>%s</strong> participated in the poll '%s'.<br/><br/>To go directly to the poll, you can use this link: <a href=\"%s\">%s</a>" : "Ciao %s,<br/><br/><strong>%s</strong> ha parteciapto al sondaggio '%s'.<br/><br/> Vai direttamente al sondaggio con il seguente collegamento: <a href=\"%s\">%s</a>",
+	"ownCloud Polls -- New Comment" : "ownCloud Polls -- Nuovo commento",
```

Comment:
ownCloud needs to be removed here. See https://github.com/nextcloud/polls/blob/master/l10n/en.json#L89-L92

Reply:
Let's see, what the robot can do.

Reply:
Merge! He will handle it, I think... :D